### PR TITLE
cleaned up the README a little

### PR DESCRIPTION
**Pull request by the872**
- renamed the variables in [Getting Started](https://github.com/pavanramkumar/pyglmnet#getting-started) to match naming convention with rest of project per [this](https://github.com/pavanramkumar/pyglmnet/pull/93/files)
- fixed the resources link that is opening the markdown version of resources

### CODE REVIEW

**Comment by jasmainak** [comment]
LGMT. +1 for merge

File: README.md [hunk 1]
```diff
@@ -52,25 +53,24 @@ beta = np.array(beta.todense())
...
+fit_param = glm[-1].fit_
 
 # we can use fitted parameters to predict
-yhat = model.predict(scaler.transform(Xt))
+yhat = glm.predict(scaler.transform(Xt))
 ```
 
-See full example on how to use `pyglmnet`
-[here](http://pavanramkumar.github.io/pyglmnet/auto_examples/plot_poisson.html)
+[See the full example on how to use `pyglmnet`](http://pavanramkumar.github.io/pyglmnet/auto_examples/plot_poisson.html)
```
Comment (jasmainak) [inline README.md]: why did you change this?

Comment (jasmainak) [inline README.md]: @pavanramkumar we should use relative paths so that the links don't break if we move where the documentations are stored.

**Comment by jasmainak** [comment]
can you also add the dependencies, that is address #69 ?

File: README.md (hunk 1 above)
Comment (pavanramkumar) [inline README.md]: @jasmainak i think it's a grammar edit! i'll raise a separate issue for relative paths

**Comment by pavanramkumar** [comment]
@th872 thanks for this! 
@jasmainak want to continue dependencies discussion in #69 so i can merge this PR?

File: README.md (hunk 1 above)
Comment (jasmainak) [inline README.md]: Ah, but the links have changed. Now you're linking the whole text instead of just "here" !

**Comment by jasmainak** [comment]
okay for doing the dependencies in a separate PR :)

File: README.md (hunk 1 above)
Comment (the872) [inline README.md]: @jasmainak good mention on the relative links, I can follow that up and fix it quickly later today. Also, I took out the 'here' link and made the whole thing a link so it's a little more visually appealing and easier to click.

Comment (jasmainak) [inline README.md]: okay, sounds great!

**Comment by jasmainak** [comment]
thanks @the872 . I merged this. I'll let you follow up on the relative links in another PR :)

**Comment by the872** [comment]
@jasmainak didn't use ci skip for just the readme commits 🙃

Sorry about that

**Comment by jasmainak** [comment]
Ah no problem :) We don't have a lot of activity anyway ;)
